HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -71,12 +84,12 @@ protected void onStart() {
 	}
 	
 	@Override
-    protected void onResume()
-    {
+	protected void onResume()
+	{
```

Comment:
nit: { should be on sameline as method. Consider setting style rules or installing an auto formatter to take care of these things.